angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/option-link.html

```diff
@@ -17,6 +16,5 @@
            <option ng-repeat='obj in portlet.widgetData[config.arrayName]' value="{{obj[config.value]}}" title='{{obj[config.display].length > 26 ? obj[config.display] : ""}}'>{{obj[config.display] | trimMiddle:26}}</option>
        </select>
     </div>
-  </div>
```

Comment:
It's not obvious to me why this `<div>` is disappearing. That might be okay -- I can't understand everything. :) Or maybe that means something interesting is happening here worth articulating?